lint/util.py

```diff
@@ -458,4 +458,4 @@ def get_sl_version():
         metadata = load_json("package-metadata.json", from_sl_dir=True)
         return metadata.get("version")
     except Exception:
-        return "unknown"
+        return VERSION
```

Comment:
Since we're talking semver here, you could make this `VERSION + "-dev"`, for example.

Reply:
Why would a plugin care if it's "-dev"? Is that really something you want a plugin to be able to switch on? Either their API breaks or it doesn't, beyond the minor version plugins shouldn't care.

Reply:
Sorry, misunderstood.